internal/hcl/modules/loader.go

```diff
@@ -72,38 +73,63 @@ func NewModuleLoader(path string, credentialsSource *CredentialsSource, logger *
 
 // downloadDir returns the path to the directory where remote modules are downloaded relative to the current working directory
 func (m *ModuleLoader) downloadDir() string {
-	return filepath.Join(m.Path, downloadDir)
+	return filepath.Join(m.cachePath, downloadDir)
 }
 
 // manifestFilePath is the path to the module manifest file relative to the current working directory
-func (m *ModuleLoader) manifestFilePath() string {
-	return filepath.Join(m.Path, manifestPath)
+func (m *ModuleLoader) manifestFilePath(projectPath string) string {
+	if m.cachePath == projectPath {
+		return filepath.Join(m.cachePath, manifestPath)
+	}
+
+	rel, _ := filepath.Rel(m.cachePath, projectPath)
+	sum := md5.Sum([]byte(rel)) //nolint
+	return filepath.Join(m.cachePath, ".infracost/terraform_modules/", fmt.Sprintf("%x.json", sum))
```

Comment:
```suggestion
	return filepath.Join(m.cachePath, ".infracost/terraform_modules/", fmt.Sprintf("manifest-%x.json", sum))
```
Should we prefix this with `manifest-` so it's clear it's a manifest file.

Reply:
good idea